### ISSUE TITLE
[Bug] ArgumentNullException when creating MsalException

### ISSUE THREAD

**Issue by tipa** [AzureAD/microsoft-authentication-library-for-dotnet#1418]
**Which Version of MSAL are you using ?**
MSAL 4.4.0

**Platform**
UWP

**What authentication flow has the issue?**
* Desktop / Mobile
    * [x] Interactive
    * [ ] Integrated Windows Auth
    * [ ] Username Password
    * [ ] Device code flow (browserless)
* Web App
    * [ ] Authorization code
    * [ ] OBO
* Web API
    * [ ] OBO

**Actual behavior**
Some of my users are experiencing a new issue that hasn't been present before. I initially got aware of it 2 weeks ago.

```
System.ArgumentNullException: Value cannot be null.
   at Microsoft.Identity.Client.MsalException.set_ErrorCode(String) + 0x4e
   at Microsoft.Identity.Client.MsalException..ctor(String, String) + 0x36
   at Microsoft.Identity.Client.MsalServiceException..ctor(String, String) + 0xc
   at Microsoft.Identity.Client.Internal.Requests.InteractiveRequest.VerifyAuthorizationResult() + 0x1db
   at Microsoft.Identity.Client.Internal.Requests.InteractiveRequest.<ExecuteAsync>d__9.MoveNext() + 0x23a
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x21
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0x70
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task) + 0x38
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task) + 0x17
   at Microsoft.Identity.Client.Internal.Requests.RequestBase.<RunAsync>d__14.MoveNext() + 0x477
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x21
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0x70
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task) + 0x38
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task) + 0x17
```

**Comment by jennyf19** [comment]
@tipa do you happen to have logs for this?

**Comment by tipa** [comment]
I don't think so, what kind of logs are you looking for or how could I collect them?

**Comment by jennyf19** [comment]
@tipa something like [this](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/logging#at-a-glance) would work, and even a repro if you have one, but sounds like it's not consistent. Do you have any information on the set-up or scenarios of the users who experienced this?

**Comment by tipa** [comment]
Unfortunately I cannot reproduce this problem locally to gather logs, it only happened to two of my users so far. What specific information would you need from those users, I could ask them via mail.

**Comment by jennyf19** [comment]
@tipa yeah..i thought that would be the case. it would be most helpful for us to know how to repro it, so anything regarding the customer set-up could help in that regard, and where they were in the auth flow when it happened...was this a new log-in, was a conditional access policy applied and they had to consent, were they asked to sign-in again, etc...

**Comment by tipa** [comment]
Alright, I will ask my users these questions and let you know here if and when they respond

**Comment by jennyf19** [comment]
This is included in the [4.5.0 release](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/releases/tag/4.5.0).
